### ISSUE TITLE
Partial Kibana screenshot in email report

### ISSUE THREAD

**Issue by ceeeekay** [sirensolutions/sentinl#468]
1. Sentinl and Kibana version: 6.2.3
Search Guard 6.2.3
Chromium-browser 66.0.3359.181-0ubuntu0.16.04.1

2. Issue description: Emailed report screenshot only contains the Kibana sidebar, not the dashboard content.

3. Reproducing the issue step-by-step: Create fault state in ES, run report, wait for email.
3a. [Raw report](https://gist.github.com/ceeeekay/556c7c8401cadcf651f98629c282a62c)

4. Expected results vs. actual results: Expect to see a screenshot of a full Kibana page but just get the left sidebar.

5. Sample of data: n/a

6. Additional details: [Kibana config](https://gist.github.com/ceeeekay/cab9f57d4cc5495511764c7c624900b2)

Brand new Sentinl user here. Sorry if this is basic or I've done something stupid here - it's my first day using Sentinl and I feel like I'm going in circles with this report issue.

I've made a very simple email watcher, which works fine. I copied the bulk of the watcher code into a new report, and attempted to get a screenshot of a Kibana dash which shows the cause of the error, but can not get it to capture the full screen. SG auth does not appear to be the problem I can see it fetching Kibana assets after the login screen.

I'm not sure if it's my Sentinl config, or possibly something to do with chromium browser.

tl;dr: Report runs, email arrives, but the screenshot is incomplete.

Any help please?

**Comment by ceeeekay** [comment]
@lmangani - perfect.  I increased the delay and I have the full page.

Thanks very much!
